pkg/configurator/methods_test.go

```diff
@@ -271,11 +271,11 @@ func TestCreateUpdateConfig(t *testing.T) {
 			},
 			updatedMeshConfigData: &v1alpha1.MeshConfigSpec{
 				Sidecar: v1alpha1.SidecarSpec{
-					InitContainerImage: "openservicemesh/init:v0.8.2",
+					InitContainerImage: "openservicemesh/init:v0.9.0-rc.1",
```

Comment:
If this test is only verifying that the value is changed to something particular and it doesn't really matter what we're updating it to, then I would argue against updating it for the same reasons I stated above.